packages/ra-data-simple-prisma/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # ra-data-simple-prisma
 
+## 5.5.0
+
+### Minor Changes
+
+- whitelist ids
+- Allow inline field perms
```

Comment:
not sure why this has sneaked in from previous release, could you remove this bit please?